language-guides/spring-data-neo4j/spring-data-neo4j.adoc

```diff
@@ -34,129 +35,194 @@ When developing with Neo4j, please use JDK 8 and your favorite IDE.
-* multiple transports: *binary protocol*, HTTP and embedded
+* multiple transports: *binary protocol*, HTTP, and embedded
 * persistence lifecycle events
 
 ==== Quickstart
 
-.Spring-Data-Neo4j Dependency
+Spring Boot takes on much of the responsibility of application configuration and bootstrap, so we have chosen to take advantage of that assistance in our project, as well.
+The example project code for https://github.com/neo4j-examples/movies-java-spring-data-neo4j[Spring Data Neo4j^] is checked in to GitHub.
+You can clone the repository and run the code along with this guide, or you can build the project from the ground up from the http://start.spring.io[Spring Initializr^] page.
+For this approach, you can follow along with a https://medium.com/neo4j/spring-into-neo4j-with-spring-data-5-spring-boot-2-and-neo4j-3962fb1ea067[blog post^] for step-by-step instructions.
+
+The dependency for Spring Data Neo4j needs to be added in order to use the capabilities.
+The rest of the dependencies set up the object-graph mapper so that we can relate our POJOs to the graph backend.
+
+.Spring-Data-Neo4j dependency
 [source,xml,subs="verbatim,attributes"]
 ----
 <dependencies>
     <dependency>
-        <groupId>org.springframework.data</groupId>
-        <artifactId>spring-data-neo4j</artifactId>
-        <version>{spring-data-neo4j-version}</version>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-data-neo4j</artifactId>
     </dependency>
 </dependencies>
 ----
 
-.Spring-Config
-[source,java]
-----
-@Configuration
-@EnableNeo4jRepositories("org.neo4j.cineasts.repository")
-@EnableTransactionManagement
-@ComponentScan("org.neo4j.cineasts")
-public class PersistenceContext {
-
-    @Override
-    public SessionFactory getSessionFactory() {
-        return new SessionFactory("org.neo4j.cineasts.domain");
-    }
-}
-----
+The entities outline our main objects in the application.
+In our graph, they will be the nodes.
+
+image::{img}GraphModel.png[]
+
+There is a `Person` entity and a `Movie` entity, so a domain class for each is needed.
+In each case, the class is annotated with `@NodeEntity` to set these objects as nodes in the graph.
+Each contains an ID field that is annotated as the `id` and populated with a value generated by the database (`@GeneratedValue`).
+After the `id`, additional fields are set up to hold different information we want to capture about the object.
+
+The last few lines of each class map the relationship between the nodes.
+Both the `Person` and `Movie` classes are annotated with `@Relationship` and name the type of relationship as `ACTED_IN`.
+The direction property is outgoing by default, so we must specify that the relationship is incoming on the `Movie` node.
+To capture additional attributes about the relationship between `Person` and `Movie`, a new entity called `Role` is created for `Person` to be connected to `Movie` by the `Role` he/she played.
```

Comment:
he/she -> they?

Reply:
Can we split this paragraph up into  the first bit which talks about the entities and the primitive fields.

And put the second part after we showed the code? so it's more obvious?

Reply:
And clearly separate out the text related to "Relationship entity"